### DISCUSSION TITLE
Edge Index Creation Error

### DISCUSSION THREAD

**Discussion by NoxideLive** [JanusGraph/janusgraph#2983] [Q&A] [answered]
Hi

I am trying to create an Edge Index

I am running the following code

```
mgmt = graph.openManagement()
gate_to = mgmt.getEdgeLabel('gate_to')
stargate_id = mgmt.makePropertyKey("stargate_id").dataType(Long.class).make()
mgmt.buildEdgeIndex(gate_to, 'GateToEdges', Direction.BOTH, Order.incr, stargate_id)
mgmt.commit()
```

And receiving the following error

```
No such property: incr for class: org.apache.tinkerpop.gremlin.process.traversal.Order
```

I have checked the JavaDoc and this static property does exist and the docs specify to use it here: https://docs.janusgraph.org/schema/index-management/index-performance/ under “Edge Indexes”

**Comment by FlorianHockmann** [comment, marked as answer]
Where do you see `Order.incr` mentioned in the docs? `Order.incr` was replaced by `Order.asc` [by TinkerPop](https://tinkerpop.apache.org/docs/current/upgrade/#_introducing_order_asc_and_order_desc) and therefore also in JanusGraph.

**Reply by NoxideLive** [reply]
It seems i was looking at old Javadoc not the current version
